sympy/stats/crv_types.py

```diff
@@ -1177,7 +1177,14 @@ def check(a, b):
     def pdf(self, x):
         a, b = self.a, self.b
         return b**a/gamma(a) * x**(-a-1) * exp(-b/x)
-
+    def sample(self):
+	try :
+	     from scipy.stats import invgamma
+	except ImportError:
+	     raise ImportError("can't import invgamma from scipy.stats")
```

Comment:
Sorry i  am not familiar with this and hence i am not sure if this is possible without scipy